src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -525,6 +525,8 @@ Note:
     <string name="revanced_speed_overlay_value_title">Speed overlay value</string>
     <string name="revanced_speed_overlay_value_summary">Speed overlay value between 0-8.0.</string>
     <string name="revanced_speed_overlay_value_invalid_toast">Speed overlay value must be between 0-8.0. Reset to default values.</string>
+    <string name="revanced_disable_zoom_overlay_title">Disable zoom overlay</string>
```

Comment:
`Disable haptic feedback` literally 'disables' haptic feedback
`Disable zoom overlay` does not

Instead of disabling the player's pinch-to-zoom behavior, patch only hides the zoom overlay

That's why `Hide zoom overlay` seems more appropriate instead of `Disable zoom overlay`